### PR TITLE
[Bot-ApplicationInsights] Fix disparities

### PR DESCRIPTION
**Pull request by Batta32**
Fix 1216
Related to 1166

## Description
We compared the structure/code migration/unit tests/documentation of the bot-applicationinsights library between [Java](https://github.com/microsoft/botbuilder-java/tree/main/libraries/bot-applicationinsights) and [C#](https://github.com/microsoft/botbuilder-dotnet/tree/main/libraries/Microsoft.Bot.Builder.ApplicationInsights) and we found disparities and fixes that this PR includes.

## Specific Changes
- Remove [test](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-applicationinsights/pom.xml#L78) scope of botbuilder package in `pom.xml`
- Send correct type of telemetry depending on the unit test
- Add missing unit test to validate when the context is null in the [onTurn](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-applicationinsights/src/main/java/com/microsoft/bot/applicationinsights/core/TelemetryInitializerMiddleware.java#L54) method
- Set timestamp to [AvailabilityTelemetry](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-applicationinsights/src/main/java/com/microsoft/bot/applicationinsights/ApplicationInsightsBotTelemetryClient.java#L87-L95) in _trackAvailability_ method

## Testing
_mvn clean install passing successfully_
image

### CODE REVIEW

File: libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java [hunk 1]
```diff
@@ -129,4 +129,13 @@ public void telemetryInitializerMiddlewareNotLogActivitiesDisabled() {
         List<String> eventNames = eventNameCaptor.getAllValues();
         Assert.assertEquals(0, eventNames.size());
 	}
+
+	@Test
+    public void telemetryInitializerMiddlewareWithUndefinedContext() {
+        // Arrange
+	    BotTelemetryClient mockTelemetryClient = Mockito.mock(BotTelemetryClient.class);
```
Comment (VictorGrycuk) [inline libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java:136]: This line has a weird indentation

File: libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java [hunk 2]
```diff
@@ -129,4 +129,13 @@ public void telemetryInitializerMiddlewareNotLogActivitiesDisabled() {
         List<String> eventNames = eventNameCaptor.getAllValues();
         Assert.assertEquals(0, eventNames.size());
 	}
+
+	@Test
+    public void telemetryInitializerMiddlewareWithUndefinedContext() {
+        // Arrange
```
Comment (VictorGrycuk) [inline libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java:135]: The test has `Arrange` but missing `Act` and `Assert`

**Review by VictorGrycuk — CHANGES_REQUESTED** [review]

File: libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java (hunk 1 above)
Comment (Batta32) [reply libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java:136]: Yes, that's weird how we see this here, but it's correct

File: libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java (hunk 2 above)
Comment (Batta32) [reply libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java:135]: Fixed

**Review by VictorGrycuk — APPROVED** [review]
